### PR TITLE
[NUI] Replace Signal with callback.

### PR DESCRIPTION
**Pull request by huayongxu**
### Description of Change ###

The related patch at csharp-binder side:
https://review.tizen.org/gerrit/#/c/platform/core/uifw/dali-csharp-binder/+/260905/

### API Changes ###

 - ACR:

### CODE REVIEW

**Review by rabbitfor — APPROVED** [review]
I only checked the code concerned with `PageLoadStarted` event.
It looks good and I think other parts are very similar to the changes in `PageLoadStated` so I'm going to approve it.

By the way, it seems that the patch is including code that are not related with signal refactoring : Binding DeleteXXX of web classes.
Please make a PR includes one subject from the next time :)

**Comment by rabbitfor** [comment]
!!Please do not merge this PR!!
Until I checked all integration process is OK with dali.
I am going to merge this after check.

**Review by zg2nets — COMMENTED** [review]
lgtm

**Pull request merged**
